src/components/picker/PickerItemsList.tsx

```diff
@@ -27,11 +28,19 @@ const PickerItemsList = (props: PickerItemsListProps) => {
     searchPlaceholder = 'Search...',
     onSearchChange,
     renderCustomSearch,
+    renderCustomDialogHeader,
     useSafeArea,
+    useDialog,
+    mode,
     testID
   } = props;
   const context = useContext(PickerContext);
   const [wheelPickerValue, setWheelPickerValue] = useState<PickerSingleValue>(context.value ?? items?.[0].value);
+  const wrapperContainerStyle = useMemo(() => {
+    const shouldFlex = Constants.isWeb ? 1 : useDialog ? 0 : 1;
+    const style = {flex: shouldFlex ? 1 : 0, maxHeight: Constants.isWeb ? Constants.windowHeight * 0.75 : undefined};
```

Comment:
WDYM by hook? 
How it would look like? 
In any case, if we want to do something like that, I wouldn't start with this PR, we can do it as a separate task